plugin/mutate/doc/design/options.md

```diff
@@ -75,6 +76,12 @@ The flags for the different commands will be listed in the following format:
     - *killedByCompiler* : Invalid mutants generated that caused the compilation of the project to fail.
     - *timeout* : Mutants that timed out during test suite execution.
 
+
+-   --id : Specify a specific mutant by MutationId.
```

Comment:
MutationId is an internal, programming terminology? What does the html report say?